flutter_secure_storage_windows/windows/flutter_secure_storage_windows_plugin.cpp

```diff
@@ -65,8 +65,8 @@ namespace
     bool ContainsKey(const std::string &key);
   };
 
-  const std::string ELEMENT_PREFERENCES_KEY_PREFIX = "VGhpcyBpcyB0aGUgcHJlZml4IGZvciBhIHNlY3VyZSBzdG9yYWdlCg_";
-  const int ELEMENT_PREFERENCES_KEY_PREFIX_LENGTH = 55;
+  const std::string ELEMENT_PREFERENCES_KEY_PREFIX = SECURE_STORAGE_KEY_PREFIX;
+  const int ELEMENT_PREFERENCES_KEY_PREFIX_LENGTH = (sizeof SECURE_STORAGE_KEY_PREFIX) - 1;
```

Comment:
Out of curiosity, why the - 1 at the length?

Reply:
In c++, sizeof returns the string length rather than the char count and this includes the null string terminator.

Reply:
Thanks for the explanation!